### PR TITLE
test: Use document.querySelectorAll instead of Sizzle

### PR DESCRIPTION
**Pull request by stefwalter**
This removes one more dependency from our list.

### CODE REVIEW

**Comment by dperpeet** [comment]
there's a symlink in avocado https://github.com/cockpit-project/cockpit/blob/master/test/avocado/sizzle.js

**Comment by dperpeet** [comment]
... and I think the tests depend on the field error message: https://github.com/cockpit-project/cockpit/pull/3533/files#diff-6d38675d5be5983315770e1e452f19a2L916
I guess we could adapt them, but this will most likely fail.

**Comment by stefwalter** [comment]
>  I guess we could adapt them, but this will most likely fail.

Well it's invalid HTML for starters ... to have two duplicate ids. check-login doesn't fail. But lets see.

**Comment by dperpeet** [comment]
> Well it's invalid HTML for starters ... to have two duplicate ids. check-login doesn't fail. But lets see.

ah ok, I didn't see the duplicate, only that you removed one.

File: test/phantom-lib.js [hunk 1]
```diff
@@ -7,11 +7,12 @@
...
 localStorage.clear();
 
-if (!window.Sizzle)
-    throw "Sizzle was not properly loaded"
-
 function ph_select(sel) {
-    return Sizzle(sel);
+    var i, els = [];
+    var list = document.querySelectorAll(sel);
+    for (i = 0; i < list.length; i++)
```
Comment (dperpeet) [inline test/phantom-lib.js]: note that according to https://developer.mozilla.org/en-US/docs/Web/API/NodeList#A_.22live.22_collection
we can also use `Array.prototype.slice.call(div_list);` here
Would that be less compatible than the simple loop?

**Comment by dperpeet** [comment]
Quite a few tests are broken with this. If we can fix the css selectors, then it would be awesome to get rid of sizzle.

File: test/phantom-lib.js (hunk 1 above)
Comment (stefwalter) [inline test/phantom-lib.js]: I tried it. It doesn't work with phantomjs.

**Pull request closed without merging**